vars/uploadToS3.groovy

```diff
@@ -0,0 +1,7 @@
+void call(Map args = [:]){
+
+    withAWS(role: "${ARTIFACT_UPLOAD_ROLE_NAME}", roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
+        s3Upload(file: args.sourcePath, bucket: args.artifactBucket, path: args.destinationOnS3)
```

Comment:
I would rename these arguments: 

- `sourcePath` looks good
- `artifactBucket` should just be `bucket`
- `destinationOnS3` should just be `path`